### PR TITLE
[web] add CanvasKit to CIPD; make it a DEPS dependency; add a manual roller script

### PR DESCRIPTION
**Pull request by yjbanov**
- Make CanvasKit a DEPS dependency that fetches CanvasKit from a CIPD package.
- Add a script for rolling CanvasKit to a new version.

This PR does not change how the engine loads CanvasKit. It is still loaded from unpkg.com by default. The loading logic will be updated in a follow-up PR.

### CODE REVIEW

File: lib/web_ui/lib/src/engine/canvaskit/initialization.dart [hunk 1]
```diff
@@ -86,7 +91,7 @@ const bool canvasKitForceCpuOnly = bool.fromEnvironment(
 /// NPM, update this URL to `https://unpkg.com/canvaskit-wasm@0.34.0/bin/`.
 const String canvasKitBaseUrl = String.fromEnvironment(
   'FLUTTER_WEB_CANVASKIT_URL',
-  defaultValue: 'https://unpkg.com/canvaskit-wasm@0.28.1/bin/',
+  defaultValue: 'https://unpkg.com/canvaskit-wasm@$canvaskitVersion/bin/',
```
Comment (hterkelsen) [inline lib/web_ui/lib/src/engine/canvaskit/initialization.dart:94]: FYI @jacob314 you will probably need to update the devtools code that reads the version from here

**Review by hterkelsen — APPROVED** [review]
LGTM

File: lib/web_ui/lib/src/engine/canvaskit/initialization.dart (hunk 1 above)
Comment (yjbanov) [reply lib/web_ui/lib/src/engine/canvaskit/initialization.dart:94]: Can devtools read it from `canvaskit_lock.yaml`?

Comment (jacob314) [reply lib/web_ui/lib/src/engine/canvaskit/initialization.dart:94]: sure that would be fine.

Comment (yjbanov) [reply lib/web_ui/lib/src/engine/canvaskit/initialization.dart:94]: Thanks for confirming!

**Pull request merged**